line_detection_updated.py

```diff
@@ -0,0 +1,302 @@
+'''
+This module creates the functions and class that are used to detect
+straight lines in astronomical images.
+
+Process
+---------
+1. Ingest the raw image
+2. Apply brightness cuts and standardize the image
+3. Perform binary thresholding on the image
+4. Blur the image
+5. Find edges
+6. Apply Hough line on the edge image
+7. Cluster the estimated lines
+8. Return all the estimated lines along with their cluster label
+
+'''
+
+# Importing necessary libraries
+import numpy as np
+import matplotlib.pyplot as plt
+
+import cv2
+
+from skimage.transform import hough_line, hough_line_peaks
+from sklearn.cluster import MeanShift
+
+# Defining functions
+
+
+def image_mask(img, percent):
+    """
+    Provides mask coordinates for an image
+
+    Parameters
+    ----------
+    img : 'numpy.ndarray'
+        The image to be masked
+    percent : `float`
+        percentage of the image borders to be masked out.
+        If x percent is provided, x/2 percent of each edge is masked out
+
+    Returns
+    --------
+    mask coordinates : `int`
```

Comment:
This isn't what the return of this function is? See `return x_dim_top, x_dim_bottom, y_dim_left, y_dim_right`.